frontend/src/lib/components/infoBadge/InfoBadge.svelte

```diff
@@ -26,4 +26,7 @@
 
 -->
 
-<div class={finalClasses}><span class="m-0 p-0">{text}</span></div>
+<div
+  class={`${baseClasses} ${disabled ? 'border-0 bg-black text-white opacity-20' : 'badge-warning'} ${classes}`}>
```

Comment:
We can do without backticks here:

`class="{baseClasses} {disabled ? 'border-0 bg-black text-white opacity-20' : 'badge-warning'} {classes}"`